modules/basegl/src/shadercomponents/timecomponent.cpp

```diff
@@ -94,9 +94,12 @@ TimeComponent::TimeComponent(std::string_view name,
 std::string_view TimeComponent::getName() const { return name_; }
 
 void TimeComponent::process(Shader& shader, TextureUnitContainer&) {
-    shader.setUniform(name_, std::chrono::duration_cast<std::chrono::duration<float, std::milli>>(
-                                 std::chrono::steady_clock::now().time_since_epoch())
-                                 .count());
+    std::chrono::duration<float, std::milli> t(0.0f);
+    if (enabled_) {
```

Comment:
Is is not enough with stopping the timer when it is disabled?

Reply:
No, `time_since_epoch()` will always return a different value every time `process()` is called. Hence the impression of an animation while for example moving the camera or triggering a re-evaluation of the corresponding processor.
Also any brushing interaction triggers an auto start of the animation.

Reply:
I see. I know that this is not exactly this PR, but it seams to be error prone to have both enabled and running. They seem very similar to me and their interaction is quite intricate (e.g `setRunning(true)` will do nothing when disabled)